docs/compose.md

```diff
@@ -67,11 +67,33 @@ FROM nvcr.io/nvidia/tritonserver:<xx.yy>-py3-min
 COPY --from=full /opt/tritonserver/bin /opt/tritonserver/bin
 COPY --from=full /opt/tritonserver/lib /opt/tritonserver/lib
 ```
+Then install dependencies outlined in [build.py](https://github.com/triton-inference-server/server/blob/main/build.py#L670-L694)
```

Comment:
Refer to function name instead of line numbers since the line numbers will quickly become out of date.